sympy/concrete/summations.py

```diff
@@ -1136,6 +1136,9 @@ def eval_sum_hyper(f, i_a_b):
             if res is not None:
                 return Piecewise(res, (old_sum, True))
         else:
+            zb_try = zb_sum(f, i_a_b)
```

Comment:
The eval_sum_hyper function is specifically for the hypergeometric algorithm. This function should be put in in the eval_sum_symbolic function.